src/Checkbox/index.js

```diff
@@ -42,14 +42,14 @@ const Checkbox = ({
       <Icon
         fontSize="24px"
         name={icon}
-        color={disabled ? 'grey20' : 'grey50'}
+        color={disabled ? 'icon.disabled' : 'icon.default'}
```

Comment:
I wonder if we should actually have a `disabled` prop on `Icon` instead that contains this. Feels painful for everywhere we have a "disabled icon" we'd need to do this similar check.

Reply:
Good point. I think I like the disabled prop better as well.

Reply:
Based on our chat, we are going to hold off on doing this change. This is specific to the icon next to the label of a checkbox and we were thinking icons as a whole.